GOVERNANCE.md

```diff
@@ -0,0 +1,145 @@
+# io.js Project Governance
+
+## Technical Committee
+
+The io.js project is jointly governed by a Technical Committee (TC)
+which is responsible for high-level guidance of the project.
+
+The TC has final authority over this project including:
+
+* Technical direction
+* Project governance and process (including this policy)
+* Contribution policy
+* GitHub repository hosting
+* Conduct guidelines
+* Maintaining the list of additional Collaborators
+
+Initial membership invitations to the TC were given to individuals who
+had been active contributors to io.js, and who have significant
+experience with the management of the io.js project. Membership is
+expected to evolve over time according to the needs of the project.
+
+For the current list of TC members, see the project
+[README.md](./#current-project-team-members).
+
+## Collaborators
```

Comment:
I intentionally put he Collaborators section here so as not to relegate them to too far below the TC and highlight that there's more than just the TC involved. Putting it after Consensus Seeking Process puts this section right at the bottom.